manifests/plugin/remote_execution/ssh.pp

```diff
@@ -0,0 +1,67 @@
+# = Foreman Proxy Remote Execution SSH plugin
```

Comment:
Yep, both .sync.yml (for modulesync) and Rakefile (to avoid having to run modulesync now).